docs/features/SupportedRFCs.md

```diff
@@ -82,8 +82,8 @@ A summary of the Aries Interop Profiles and Aries RFCs supported in ACA-Py can b
 | Multi use invitations            | :white_check_mark:  |         |
 | Invitations using public did     | :white_check_mark:        |         |
 | Implicit pickup of messages in role of mediator | :white_check_mark:        |         |
-| [Revocable Indy Credentials](https://github.com/hyperledger/indy-hipe/tree/main/text/0011-cred-revocation) | :white_check_mark:        |         |
-| Multi-Tenancy      | :white_check_mark:        | [Documentation](https://github.com/hyperledger/aries-cloudagent-python/blob/main/Multitenancy.md) |
+| [Revocable Indy Credentials](https://github.com/hyperledger/indy-hipe/tree/1.0.0-rc2/text/0011-cred-revocation) | :white_check_mark:        |         |
```

Comment:
I don't think this repo, `hyperledger/indy-hipe`, will have an `1.0.0-rc2` branch.